.github/workflows/test_hipblaslt.yml

```diff
@@ -34,8 +34,6 @@ jobs:
   test_hipblaslt:
     name: "hipBLASLt math-lib test"
     runs-on: ${{ inputs.test_runs_on }}
-    # Currently, this test is not working with Windows. This test will be enabled once this library has been enabled
-    if: ${{ inputs.platform == 'linux' }}
```

Comment:
@geomin12 how do I test this? The PR didn't run any Windows tests by default. I just added the `gfx115X-windows` label but no new jobs were triggered.

Can you point me to some docs like https://iree.dev/developers/general/contributing/#ci-behavior-manipulation for TheRock? Looking at those docs again, they don't mention the triggering - changes to git trailers affecting enabled jobs in that project also require pushing a new commit. Is that the case here too? I'd like to run new jobs without having to re-run old jobs.